### ISSUE TITLE
[Sourcing-2][Accessibility: ACC-264.1][WIP] Table: Search results not conveyed.

### ISSUE THREAD

**Issue by JyotiKumari1507** [SAP/fundamental-ngx#11272]
#### Is this a bug, enhancement, or feature request?
Bug

#### Describe your proposal.
Component having issue: https://sap.github.io/fundamental-ngx/#/platform/table/basic
On entering any value in the search field of Table's toolbar and press enter key, item get filtered based on the entered value but the same is not conveyed to the screen reader users. Search results should be conveyed to the user.

#### Can you handle that on the application side
No

#### Which versions of Angular and Fundamental Library for Angular are affected? Please, specify the exact version. (If this is a feature request, use current version.)
All latest version

#### If this is a bug, please provide steps for reproducing it; the exact components you are using;
Using Screen reader can check by implementing similar case. Just use table with search on toolbar and enter any value in the search field of Table's toolbar and press enter key, item get filtered based on the entered value but the same is not conveyed to the screen reader users. 

#### In case this is Accessibility related topic, did you consult with an accessibility expert? If not, please do so and share their recommendations. 
Yes we have got this bug from Accessibility expert only.

#### Did you check the documentation and the API?
Yes

**Comment by droshev** [comment]
@JyotiKumari1507 Can you provide more input, steps, example, and JAWS history?

**Comment by JyotiKumari1507** [comment]
@droshev  We have search functionality in Plaform Tables. When we search in that table, based on search row get shortlisted  But same is not conveyed to screen reader. Accessibility team expect search result should read out by screen reader. We can have more discussion in next call

**Comment by droshev** [comment]
@JyotiKumari1507 we need the input written so it will be easier to process

**Comment by JyotiKumari1507** [comment]
@droshev Attaching screen reader screenshot for more details:
On entering any value in the search field and pressing  enter key, applications get filtered based on the entered value but the same is not conveyed to the screen reader users.

<img width="903" alt="image" src="https://github.com/SAP/fundamental-ngx/assets/157268666/876e0a0b-e561-4ae0-8f22-7f89dc33eabd">

**Comment by droshev** [comment]
Most of the fields are not filled.

**Comment by JyotiKumari1507** [comment]
@droshev can you pls let me know what input field expecting from us?

**Comment by droshev** [comment]
@JyotiKumari1507 Can you create a stackblitz example, steps to reproduce and what the expectation is?

**Comment by droshev** [comment]
@JyotiKumari1507 Can you create a stackblitz example, steps to reproduce and what the expectation is? otherwise we will close the ticket.

**Comment by divyamalhotra488** [comment]
Hi @droshev, the expectation is that the results of a 'Search' in the table toolbar should be conveyed using screen reader.
Currently, search is able to filter items on UI, but the same is not conveyed using screen reader. You can refer the attached screenshot above.
[Stakcblitz example](https://stackblitz.com/edit/prywkj?file=package.json)

**Comment by droshev** [comment]
In the latest version you can add `searchFieldAriaLabel` and updated it based on what the user enter. I guess that would be enough?

**Comment by droshev** [comment]
@divyamalhotra488 you can find it in the API https://fundamental-ngx.netlify.app/#/platform/table/api

**Comment by droshev** [comment]
@divyamalhotra488 you can consult that with the Accessibility experts and the specification
